app/controllers/orders_controller.rb

```diff
@@ -15,6 +15,10 @@ def create
     end
   end
 
+  def my_rewards
```

Comment:
this action's name looks a bit weird (we are in the orders controller, but the name has a reward in it) - I would consider naming it 'my orders', or something similar